docs/faq.md

```diff
@@ -34,9 +34,10 @@ How do I deal with my log-in page?
 
 If your app needs log-in, there are a couple ways to deal with it. If your login
 page is not written with Angular, you'll need to interact with it via 
-unwrapped webdriver, which can be accessed like `browser.driver.get()`. 
+unwrapped webdriver, which can be accessed like `browser.driver.get()`. You can also use
+`browser.ignoreSynchronization`, as explained in [this StackOverflow answer](http://stackoverflow.com/a/23198865/264229)
```

Comment:
Hah, we should probably have our own docs. But I'm fine with this for now, since I can't think off the top of my head where it would best live.